### PR TITLE
Layer style selecting

### PR DESCRIPTION
**Pull request by jheijari**
Layer style select to selected layers view.
Custom style button for wfs layers.

### CODE REVIEW

File: bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx [hunk 1]
```diff
@@ -88,31 +122,58 @@ export const LayerBox = ({ layer, index }) => {
...
                                     </ColAuto>
                                     <ColAuto>
-                                        <StyledNumberInput
-                                            min={0}
-                                            max={100}
-                                            value={slider}
-                                            onChange={handleOpacityChange}
-                                            formatter={value => `${value} %`}
-                                        />
+                                        <StyleLabel>{ locale.layer.styles.title }</StyleLabel>
```
Comment (ZakarFin) [inline bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx]: The style section should hidden if the layer has less than 2 styles (user can't select anything). An exception to this is layer that has the ownStyle tool (namely WFS-layers).

File: bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js [hunk 2]
```diff
@@ -45,8 +45,13 @@ class UIService extends StateHandler {
         const toDataPosition = (layers.length - 1) - toPosition;
         this.sandbox.postRequestByName('RearrangeSelectedMapLayerRequest', [layer.getId(), toDataPosition]);
     }
+
+    changeLayerStyle (layer, styleName) {
+        this.sandbox.postRequestByName('ChangeMapLayerStyleRequest', [layer.getId(), styleName]);
```
Comment (ZakarFin) [inline bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js]: We should also listen to AfterChangeMapLayerStyleEvent and update the selected value in the dropdown based on it if the style is changed by some other functionality.

Comment (jheijari) [reply bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js]: We are listening to it in `LayerViewTabsStateHandler`.

Comment (ZakarFin) [reply bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/SelectedLayersHandler.js]: Excellent!

File: bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx (hunk 1 above)
Comment (jheijari) [reply bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx]: Fixed the logic and moved style settings to it's own file.

Comment (ZakarFin) [reply bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx]: Moving this to own component makes the code a pleasure to read 👍

**Pull request merged**